Mergin/create_project_wizard.py

```diff
@@ -115,7 +115,7 @@ def initializePage(self):
 
     def populate_namespace_cbo(self):
         if self.parent.workspaces is not None:
-            for ws in self.parent.workspaces:
+            for ws in sorted(self.parent.workspaces, key=lambda x: x["name"].lower()):
```

Comment:
Code wise look fine however the sort is currently alphanumeric where I think users will expect a natural sort

So for example if project names in a workspace look like this
```
project 0
project 1
project 2
project 3
project 4
project 5
project 6
project 7
project 8
project 9
project 10
project 11
project 12
```

it will be actually sorted as
```
project 0
project 1
project 10
project 11
project 12
project 2
project 3
project 4
project 5
project 6
project 7
project 8
project 9
```

I'm not sure how often users name their projects numerically but we can implement a natural in python like this https://stackoverflow.com/a/4836734

Reply:
Thanks for pointing this out. However, as stated previously, this PR is mainly for support agents who deal with dozens of workspaces and we do not care whether the sorting is natural or not :-)

Normally, users have just a couple of them at most and I doubt their names have numbers.

Reply:
Okay then it looks good to me  :+1: